en/flight_modes_fw/mission.md

```diff
@@ -0,0 +1,372 @@
+   Enter the time to wait after landing before disarming the vehicle.
+   :::
+
+Missions can be paused by switching out of mission mode to any other mode (such as [Hold mode](../flight_modes/hold.md) or [Position mode](../flight_modes/position_mc.md)), and resumed by switching back to mission mode.
+If the vehicle was not capturing images when it was paused, on resuming it will head from its _current position_ towards the same waypoint as it as was heading towards originally.
+If the vehicle was capturing images (has camera trigger items) it will instead head from its current position towards the last waypoint it travelled through (before pausing), and then retrace its path at the same speed and with the same camera triggering behaviour.
+This ensures that in survey/camera missions the planned path is captured.
+A mission can be uploaded while the vehicle is paused, in which which case the current active mission item is set to 1.
+
+:::note
+When a mission is paused while the camera on the vehicle was triggering, PX4 sets the current active mission item to the previous waypoint, so that when the mission is restarted the vehicle will retrace its last mission leg.
+In addition, PX4 stores the last applied mission items for speed setting and camera triggering (from the already covered mission plan), and re-applies those settings on resuming the mission.
+:::
+
+:::warning
+Ensure that the throttle stick is non-zero before switching to any RC mode (otherwise the vehicle will crash).
+We recommend you centre the control sticks before switching to any other mode.
+:::
+
+For more information about mission planning, see:
+
+- [Mission Planning](../flying/missions.md)
+- [Plan View](https://docs.qgroundcontrol.com/master/en/PlanView/PlanView.html) (_QGroundControl_ User Guide)
+
+## Mission Feasibility Checks
+
+PX4 runs some basic sanity checks to determine if a mission is feasible when it is uploaded, and when the vehicle is first armed.
+If any of the checks fail, the user is notified and it is not possible to start the mission.
+
+A subset of the most important checks are listed below:
+
+- First mission item too far away from vehicle ([MIS_DIST_1WP](#MIS_DIST_1WP))
+- Any mission item conflicts with a plan or safety geofence
+- More than one land start mission item defined ([MAV_CMD_DO_LAND_START](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_LAND_START))
+- A fixed-wing landing has an infeasible slope angle ([FW_LND_ANG](#FW_LND_ANG))
+- Land start item (`MAV_CMD_DO_LAND_START`) appears in mission before an RTL item ([MAV_CMD_NAV_RETURN_TO_LAUNCH](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_RETURN_TO_LAUNCH))
+- Missing takeoff and/or land item when these are configured as a requirement ([MIS_TKO_LAND_REQ](#MIS_TKO_LAND_REQ))
```

Comment:
These all correct for FW? Any more we know of?

Reply:
no that looks complete.